src/main/java/com/minecolonies/rankup/modules/purchase/command/BuyCommand.java

```diff
@@ -0,0 +1,110 @@
+package com.minecolonies.rankup.modules.purchase.command;
+
+import com.minecolonies.rankup.internal.command.RankupSubcommand;
+import com.minecolonies.rankup.util.RankingUtils;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.CommandElement;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.event.cause.EventContext;
+import org.spongepowered.api.service.economy.account.UniqueAccount;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.action.TextActions;
+import org.spongepowered.api.text.format.TextColors;
+import org.spongepowered.api.util.Color;
+import org.spongepowered.api.util.annotation.NonnullByDefault;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)